pkg/cli/cmds/cert.go

```diff
@@ -0,0 +1,48 @@
+package cmds
+
+import (
+	"io/ioutil"
+
+	"github.com/rancher/k3s/pkg/cli/cert"
+	"github.com/rancher/k3s/pkg/cli/cmds"
+	"github.com/rancher/rke2/pkg/rke2"
+	"github.com/urfave/cli"
+)
+
+var k3sCertFlags = map[string]*K3SFlagOption{
```

Comment:
```suggestion
var k3sCertFlags = map[string]*K3SFlagOption{
```
Should this be `rke2CertFlags`?

Reply:
its reference to the flags in k3s, thats why its named k3s, I actually copied what you did for snapshot command